### ISSUE TITLE
Quad Gate Order

### ISSUE THREAD

**Issue by crife-pact** [cytoflow/cytoflow#295]
### DESCRIPTION:
  - What were you trying to do? Applying a quad gate and looking at resulting sample assignments
  - What happened? Quads are not labeled in a clockwise direction as described in the docs
  - What did you expect to happen? Quads should be clockwise in order
  
**Don't forget to attach the log file to this bug report!** 

No error log; here is the code and output.

```
tube1 = flow.Tube(file = 'data/RFP_Well_A3.fcs',
                  conditions = {'Dox' : 10.0})
tube2 = flow.Tube(file='data/CFP_Well_A4.fcs',
                  conditions = {'Dox' : 1.0})

import_op = flow.ImportOp(conditions = {'Dox' : 'float'},
                          tubes = [tube1, tube2])

ex = import_op.apply()

quad = flow.QuadOp(name = "Quad",
                            xchannel = "V2-A",
                            xthreshold = 100,
                            ychannel = "Y2-A",
                            ythreshold = 1000)


ex2 = quad.apply(ex)
qv = quad.default_view(huefacet = "Quad",
                        xscale = 'log',
                        yscale = 'log').plot(ex2)

ex2.data.groupby('Quad').size()
```

Quad
Quad_1    1783
Quad_2    2584
Quad_3    8236
Quad_4    7397
dtype: int64

![quads](https://user-images.githubusercontent.com/69477994/107711405-733c2c80-6c7c-11eb-97c0-b6fe3d89dae8.png)

**Comment by kristjaneerik** [comment]
Looks like the code indeed sets the lower left to be 3 (https://github.com/bpteague/cytoflow/blob/master/cytoflow/operations/quad.py#L220) and the lower right to be 4 (https://github.com/bpteague/cytoflow/blob/master/cytoflow/operations/quad.py#L235).

**Comment by bpteague** [comment]
Docs are wrong (sorry!!) -- as it says in the code, the gate numbers are chosen to match FACSDiva. They're arbitrary.

**Comment by bpteague** [comment]
Fixed (changed the docs, not the code) in 5264519f2b904670f03c7108bbb239f1c798457a
